docs/content/getting-started/getting-help.mdx

```diff
@@ -1,84 +1,271 @@
-
-> Is it possible to set up Dagster to automatically update an entire downstream table every time that a specific partition of an upstream table is updated? We have a table that's partitioned by customer, and we have a specific analysis that we want to do on a specific customer.
-
-> The background here is that that we've set up Fivetran to pulls in our per-customer Salesforce tables into one big Snowflake table. We have about 500 customers, and they get pulled in usually daily, but it depends on the customer. For about 20 of these customers, we have custom logic, implemented in SQL with dbt, that performs analyses that are specific to those customers. We want those analyses to run automatically when customer data is updated, so that our sales representatives can discover red flags when they check their Looker dashboards.
-
-Below is an example of a question that doesn't provide enough detail for the reader to answer it:
-
-> How do I use tables with specific upstream partitions?
-
-Below is an example a question that's difficult to digest. It requires the reader to understand an entire data pipeline to be able to answer the specific question:
+      ```python    ## specifies language
+      from dagster import ...
+      ...
+      ```
 
-> We've set up Fivetran to pulls in our per-customer Salesforce tables into one big Snowflake table. We have about 500 customers, and they get pulled in usually daily, but it depends on the customer. For about 20 of these customers, we have custom logic, implemented in SQL with dbt, that performs analyses that are specific to those customers. We want those analyses to run automatically when customer data is updated, so that our sales representatives can discover red flags when they check their Looker dashboards. How can I get this working with Dagster?
+### Bugs
+
+When filing a [GitHub issue](https://github.com/dagster-io/dagster/issues) for a bug:
+
+- **DO** include the minimum-sized working code snippet that reproduces the issue
+- **DO** include the steps you took to create the issue, if the issue occurred in the Dagster UI
+- **DO** include the Dagster version being used, if applicable
+- **DON'T** include any sensitive information
+
+### Ask questions that get answers
+
+When posting questions in [GitHub Discussions](https://github.com/dagster-io/dagster/discussions) or [Slack](https://dagster.io/slack), we recommend:
+
+- Being clear, specific, and as brief as possible
+- Not including lots of irrelevant details
+
+Generally, the less time and effort it takes for someone to digest your question, the more likely it will get answered. Check out the following table for some examples of well-phrased questions and those that could use some improvement.
+
+<table
```

Comment:
I'm kind of on the fence about how these questions are formatted, but wanted feedback.

Reply:
it took a while for me to internalize what qualifies a good question - i feel it's a lot going on with the bold and italic fonts.

how about this? https://gist.github.com/yuhan/fbd8164abfccef62ee0c750a215a1e5f

Reply:
I dig this, thank you!